CodeStarterCamp_Week2/main.swift

```diff
@@ -6,8 +6,8 @@
 //
 import Foundation
 
-let myLottoNumbers: [Int] = [1,2,3,4,5,6]
-let correctNumbers = compareToMachine(user: myLottoNumbers, machine: createLotto())
-
-printLottoResult()
+var lottoDictionary = ["1회차": [1,2,3,4,5,6]]
+var count = 1
 
+addLotto(number: 4)
+findLotto(number: 2)
```

Comment:
이제 2개의 var 전역 변수는 필요하지 않아요!

로또 티켓 5개를 딱 사고, 그걸 `let` 프로퍼티(lottoTickets)로 받아주고
그 프로퍼티를 show 메서드의 파라미터로 집어 넣는 거죠.
간단하죠? 😄 

사용자는 파라미터 숫자만 바꿔주면, 로또를 몇 장 살거고, 거기서 몇 번째 티켓을 보여달라고 할 건지
`직관적으로` 요청할 수 있겠죠. (전역변수를 바꿔가며 실행하는 것보다는요)

```suggestion
// 5개의 로또 티켓을 구매한다.
let lottoTickets = buyLottoTicket(count: 5)

// 그 5개의 로또 티켓을 파라미터로 넣고, 2번째 티켓을 print 한다.
showLottoTicket(number: 2, in: lottoTickets)
```